main.py

```diff
@@ -0,0 +1,22 @@
+"""
+Week 8
+This is to test how many hours a grocery store is open a week
+"""
+
+class Grocery_Store:
```

Comment:
Class names in python use PascalCase instead of snake_case like everywhere else: `GroceryStore`

`-1pt`